drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/window/WindowResizer.java

```diff
@@ -70,17 +70,19 @@ private void resizeWindow(DroneEvent event) {
         WebDriverConfiguration configuration = context.get(dronePoint).getConfigurationAs(WebDriverConfiguration.class);
         Validate.stateNotNull(configuration, "WebDriver configuration must not be null");
 
-        if (configuration.getDimensions() != null) {
-            String dimensions = configuration.getDimensions().toLowerCase().trim();
-            Matcher m = DIMENSIONS_PATTERN.matcher(dimensions);
+        String browser = configuration.getBrowser().toLowerCase();
+        if(!browser.equals("chromeheadless")) {
```

Comment:
Good job with this investigation. 
If both PhantomJs and HtmlUnit solves this problem using specific dimensions, then (technically) we could provide the same functionality also for Chrome headless - as you did. Then leave it as it is now - using the specific dimensions when fullscreen is used. Just log info about it...